### DISCUSSION TITLE
[BUG]:小窗口打开为什么默认是锁定的.可以进行设置吗

### DISCUSSION THREAD

**Discussion by 188083235** [magicdawn/Bilibili-Gate#126] [Q&A]
### 环境

浏览器: (如 Chrome)
浏览器版本: (如 113.0.5672.126)
脚本管理器: (如 暴力猴 / TamperMonkey)
脚本版本: (如 0.14.0, 请先更新到最新版本!!!)

### 问题详细描述

[BUG]:小窗口打开为什么默认是锁定的.可以进行设置吗

**Comment by magicdawn** [comment]
设置项太多了, 不想加了......

常用的情况, 自动播放 + 自动网页全屏, 应该直接看就行了...什么情况需要手动解锁

**Comment by owwkmidream** [comment]
#139
